website/pages/docs/secrets/databases/hanadb.mdx

```diff
@@ -73,8 +73,8 @@ the proper permission, it can generate credentials.
     lease_id           database/creds/my-role/2f6a614c-4aa2-7b19-24b9-ad944a8d4de6
     lease_duration     1h
     lease_renewable    true
-    password           8cab931c-d62e-a73d-60d3-5ee85139cd66
-    username           v-vaultuser-e2978cd0-
+    password           sU4zWnITaABEP-Zwy4sF
+    username           v-vaultuser-my-role-jQbCLE6P2VtgsqPBXK0m-1602541873
```

Comment:
I think hanadb is one that doesn't allow hyphens in usernames?
```suggestion
    username           v_vaultuser_my_role_jQbCLE6P2VtgsqPBXK0m_1602541873
```

Reply:
:+1: